app/lib/package/backend.dart

```diff
@@ -679,7 +684,7 @@ class PackageBackend {
     final guid = createUuid();
     _logger.info('Starting semi-async upload (uuid: $guid)');
     final object = tmpObjectName(guid);
-    await data.pipe(_bucket.write(object));
+    await data.pipe(_incomingBucket.write(object));
```

Comment:
I guess this is fine... but separately it would be probably be great if we got rid of the `upload` method..

Testing should just upload the fake bucket server, right?

Reply:
I'll remove it in a subsequent PR. It is used only in tests anyway...